accepted/00000-hub-reporting.md

```diff
@@ -70,26 +56,58 @@ from the main Uyuni Server database.
 Data which belong to the traditional stack only (like osad status) should not be made available in the
 reporting database as the traditional stack will be deprecated soon.
 
+# Installation
+The reporting tools will be installed by default; all the upgrade to a newer version, will install the 
+reporting tools as well.
 
-## Adding Data to the Reporting DB
+We provide a new package installed in the hub system:
+- to setup the reporting DB schema,
+- to setup and manage the reporting database,
+- to setup and manage DBs in all Uyuni Server.
+
+The package will also setup a taskomatic job to retrieve informations from all the other SUSE Manager Server
+database.
+All the other information required (e.g. Uyuni Server info) will be retrieved by already existing configuration.
+
+## Database management: setup and user management
+
+### Uyuni Hub
+On the hub side, the management tools should support:
+- Database setup and schema initialization
+- Database schema migration
+- Simple User management. The default setup will create a Read/Write user, to manage the database and write the data from into the Reporting DB. This user and the DB connection parameters are written into `/etc/rhn/rhn.conf` similar to the default DB options.
+
+### Uyuni Server
+On the server side, during installation, hub should create its own read-only user on the Reporting Databases of the single Servers. 
+This user is used by the hub to read data from the Uyuni Server DB.
+As the Servers are managed with salt, we will write a state to create an account.
+The username and password are generated on the Hub and provided as pillar data.
+On the server the state take care of the existance of the account and the Hub can store the parameters
+in its database under the system entry. 
```

Comment:
Maybe we should better state, that Hub and Single Server is the same and only name real differences?

Reply:
Same as above, I ll try to rephrase it to made more clear the single-server scenario